### DISCUSSION TITLE
Multi Monitor split across multiple windows

### DISCUSSION THREAD

**Discussion by AnorakMan** [FreeRDP/FreeRDP#10212] [Q&A]
I have 2 monitors. I can get remote sessions to start using both monitors, and that works fine.
Is it possible to have each "monitor" show in it's own X window?

**Comment by akallabeth** [comment]
I assume you´re using `xfreerdp`?
short answer: no
longer one: if you use `Xephyr` or similar you might have a chance of creating a setup that will allow this (not dynamic though)

**Comment by AnorakMan** [comment]
I stared off using Remmina, but then switched to xfreerdp cli to try and get it working. I'll look at Xephyr, thank you.
